{
  throwIfRetryRequest(caught);
  super.doUnexpectedFailure(caught);
}
